package.json

```diff
@@ -1,6 +1,17 @@
 {
-    "name": "framework-arduinopico",
-    "description": "Arduino Wiring-based Framework (RPi Pico RP2040)",
-    "url": "https://github.com/esrlephilhower/arduino-pico",
-    "version": "0.9.2"
-}
+  "name": "framework-arduinopico",
+  "version": "1.10902.0",
```

Comment:
Version is auto-generated by the package script.  It's a minor change I can do to change the #, but what's wrong with the existing `#.#.#` format?  I always update it (during package/release, but not in git) for releases, so it will be unique no matter what...

Reply:
The format I used is used by pretty much PlatformIO packages, decodable per [this](https://community.platformio.org/t/find-the-mbed-version-which-was-used-before-a-backup/16159/4?u=maxgerhardt), aka `<major package version>.<semver encoding of exact version>.datecode`, so this is just for the sake of being in-sync with PlatformIO. PlatformIO has no problems also decoding `0.9.2` if it detects that it's not in PlatformIO's preferred format. Althoug even PlatformIO has [inconsistencies](https://github.com/platformio/platform-raspberrypi/blob/128a0826e1783a9103373c2878dbc43ebc0e7f88/platform.json#L28-L57).. (see `framework-arduino-mbed`).

Note that if changed, also the [`platform.json`](https://github.com/maxgerhardt/platform-raspberrypi/blob/77e0d3a29d1dbf00fd3ec3271104e3bf4820869c/platform.json#L46-L51) must be adapted (no big deal since no package version has been released anyway)